ios/versioned-react-native/ABI38_0_0/Expo/EXFacebook/ABI38_0_0EXFacebook.podspec

```diff
@@ -18,6 +18,6 @@ Pod::Spec.new do |s|
 
   s.dependency 'ABI38_0_0UMCore'
   s.dependency 'ABI38_0_0UMConstantsInterface'
-  s.dependency 'FBSDKCoreKit', '~> 5.8.0'
-  s.dependency 'FBSDKLoginKit', '~> 5.8.0'
+  s.dependency 'FacebookSDK/CoreKit', $FacebookSDKVersion || '9.0.1'
+  s.dependency 'FacebookSDK/LoginKit', $FacebookSDKVersion || '9.0.1'
```

Comment:
I wonder why are we using some variables for native dependencies instead of just plain string versions?
Both for iOS and Android? 🤔

Reply:
To allow the developers to change the version that we use by default. Using version ranges would probably be fine, but we don't do all the QA on every possible version of these dependencies. This way is probably better if someone wants `expo-facebook` to use different version of FacebookSDK, because you explicitly opt in and you know what you're doing (forcing the module to use different dependency version as it was tested and used against).

Also keep in mind that CocoaPods throws an error when two modules want to use the same dependency but with non-matching version ranges.